src/reportSources/legacyFflogs/eventAdapter/translate.ts

```diff
@@ -123,16 +122,18 @@ export class TranslateAdapterStep extends AdapterStep {
 		}
 	}
 
-	private adaptSnapshotEvent(event: DamageEvent | HealEvent): Array<Events['snapshot' | 'actorUpdate']> {
+	private adaptFFLogsEffectEvent(event: DamageEvent | HealEvent): Array<Events['execute' | 'damage' | 'heal' | 'actorUpdate']> {
 		// Calc events should all have a packet ID for sequence purposes. Let sentry catch outliers.
 		const sequence = event.packetID
 		if (sequence == null) {
-			throw new Error('Calculated damage event encountered with no packet ID.')
+			if (event.type === 'damage') { return this.adaptDamageEvent(event) }
+			if (event.type === 'heal') { return this.adaptHealEvent(event) }
```

Comment:
comment on this redirection to damage/heal is probably worthwhile (i.e. something about damage/heal without a sequence means it's a status which doesn't get executes, etc)